modules/contenta_enhancements/contenta_enhancements.module

```diff
@@ -21,3 +48,300 @@ function contenta_enhancements_entity_base_field_info_alter(&$fields, EntityType
     $fields['title']->setDisplayConfigurable('view', TRUE);
   }
 }
+
+/**
+ * Implements hook_page_attachments().
+ */
+function contenta_enhancements_page_attachments(array &$page) {
+  $page['#attached']['library'][] = 'contenta_enhancements/fix-2886816';
+  $page['#attached']['library'][] = 'contenta_enhancements/fix-2887220';
+}
+
+/**
+ * Implements hook_local_tasks_alter().
+ */
+function contenta_enhancements_local_tasks_alter(&$local_tasks) {
+  $local_tasks['entity.oauth2_client.collection']['title'] = '📱 Clients';
+  $local_tasks['entity.oauth2_client.collection']['base_route'] = 'entity.user.collection';
+  $local_tasks['entity.oauth2_client.collection']['weight'] = 90;
+
+  $local_tasks['entity.oauth2_token.collection']['title'] = '🔑 Tokens';
```

Comment:
I'm OK with that. I want to keep it consistent with Reservoir to start talking about abstracting the common bits.